sqlite3_test.go

```diff
@@ -1091,7 +1096,7 @@ func TestQueryer(t *testing.T) {
 		}
 		n = n + 1
 	}
-	if n != 3 {
+	if n != 4 {
```

Comment:
Why 4?

Reply:
It is a 1-indexed loop, so n is 4 when its looped three times. We could insert 0,1,2 and start at 0 instead, that way n would be 3 when its done. If you prefer that I'll make that change.

Reply:
I think it would be more intuitive to have `n` be the number of rows that were returned, in which case it should start at `0`. Whether you change it to insert 0, 1, 2, or change it to compare against `n+1` is fine to me.